### ISSUE TITLE
[Adaptive Cards- Schema Explorer] : Cards and other tree items are not accessible through the keyboard keys. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9029]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)

**Pre-Requisite:**
Navigate through keyboard only.

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Schema Explorer' option and press enter key to activate it.
3. Press tab key and navigate to the Cards tree items and observe the issue.
 
**Actual Result:**
While navigating the schema explorer using tab key, Cards and other tree items are not accessible through the keyboard. Also on pressing down arrow keyboard focus is not landing on the items present under it. Also, all tree items are accessible through the tab key,

**Expected Result:**
Cards and other tree items should be accessible with keyboard and when the focus is on it then on pressing right arrow key tree item should be expanded and on pressing left arrow key it should be collapsed.
Also, on pressing down or up arrow key items present under the tree items should be navigable.

**Note:**
1. It should be working same as it is working on the below URL.
https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-1a/

2. This issue is repro throughout the left pane.

**User Impact:**
It will impact the keyboard user if the tree items are not accessible through the keyboard as they will not be able to perform desired action.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/keyboard

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Cards Tree items are not accessible through the keyboard](https://github.com/user-attachments/assets/011b0c88-548d-4023-9705-e9a13923234d)

https://github.com/user-attachments/assets/77ac2557-6b19-4b6d-af1f-00e4b57dfdcd

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
We have released another version of the website, and this fix is available. Thanks!
